src/protocols/bdx/BdxMessages.h

```diff
@@ -32,6 +32,8 @@
 namespace chip {
 namespace bdx {
 
+constexpr uint16_t kMaxFileDesignatorLen = 0xFFFF;
```

Comment:
This seems a very huge number, 65536 bytes, I see two char buffers being allocated in BdxOtaSender.h and OTARequestor.h and these are very big allocations for an embedded device. Spec mention that file designator length is two byte number. But, can we use lesser values?

Reply:
Yeah, I was using the file designator length in the spec but these are huge. I can lower it.